CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+#How to contribute
+Alfresco's maturity comes from the contributions of its community. Contributing back to Alfresco can be a bit confusing as Alfresco is a collection of separate open source projects. Each open source project officially sponsored by Alfresco has a page in this space detailing its governance policy and the method for reporting issues and submitting contributions.
+ 
+For the details on the Records Management project see[here.](https://community.alfresco.com/docs/DOC-6387-project-overview-records-management)
+
+##Getting started
+What you need to get started:
+
+* A [Jira](https://issues.alfresco.com/jira/projects/ALF/issues/?filter=allopenissues) account,
+
+* A [GitHub](https://github.com/) account
+
+Useful but not necessary:
+
+* An [Alfresco community](https://community.alfresco.com/) account - this is a good place to ask questions and find answers. 
+##Making changes
+
+We use Jira to track issues if you are committing a fix for a raised issue please include the ticket number in both the merge requests and the git commit messages.
```

Comment:
Missing full stop (or semicolon?): "We use Jira to track issues. If you are..."

Reply:
My bad, changing this now.